src/openforms/formio/tests/validation/test_password.py

```diff
@@ -0,0 +1,77 @@
+from django.test import SimpleTestCase
+
+from openforms.typing import JSONValue
+
+from ...typing import Component
+from .helpers import extract_error, validate_formio_data
+
+
+class PasswordFieldValidationTests(SimpleTestCase):
+    def test_valid_required_passwordfield(self):
+        component: Component = {
+            "type": "password",
+            "key": "foo",
+            "label": "Foo",
+            "validate": {"required": True},
+        }
+
+        data: JSONValue = {"foo": "test"}
+
+        is_valid, errors = validate_formio_data(component, data)
+
+        self.assertTrue(is_valid)
+        self.assertDictEqual(errors, {})
```

Comment:
```suggestion
        self.assertEqual(errors, {})
```
python itself figures out which more specific type assertion it needs to use, always use `self.assertEqual`